Cabal/Cabal.cabal

```diff
@@ -251,11 +257,20 @@ library
     build-depends: directory >= 1.2 && < 1.3,
                    process   >= 1.1.0.2  && < 1.5
 
+  build-tools:
```

Comment:
Well, `cabal install alex` will do the trick in a pinch. Won't even be that slow if cached. hvr's PPA has the stuff for Linux though.